### DISCUSSION TITLE
grafana拿不到数据

### DISCUSSION THREAD

**Discussion by limh720** [shizunge/endlessh-go#135] [General]
在prometheus能采集到数据
但是在grafana用15156，无法展示数据，grafana的版本是10.0.3

**Comment by shizunge** [comment]
Can your other dashboards obtain data correctly?
I am suspecting the data source is not setup correctly or when loading the dashboard, you did not select the correct data source.

**Reply by limh720** [reply]
同样的数据库，监控Node或者centos，是能展示数据的。数据源应该是能问题，我也能打开2112端口，采集到数据。

**Reply by shizunge** [reply]
Are you able to try Grafana version 10.3.0?
